src/images/zookeeper.rs

```diff
@@ -0,0 +1,90 @@
+use crate::core::Port;
+use crate::{Container, Docker, Image, WaitForMessage};
+use std::collections::HashMap;
+
+const CONTAINER_IDENTIFIER: &str = "zookeeper";
+const DEFAULT_TAG: &str = "3.5.8";
```

Comment:
Why not the latest tag `3.6.2`?

Reply:
i just thought  in kafka compatibility, i was wrong, we can specify  the version later :)